File: kildey004/programacaobasica/questao14.py
print("Calendario de acidentes")

dia = int(input("Digite a quantidade de dias sem acidentes: "))

ano = int(dia / 360)
mes = int((dia - ano * 360) / 30)
dias = int(dia - ano * 360 - mes * 30)

print("Tempo sem acidentes: " + str(ano) + " anos, " + str(mes) + " meses, " + str(dias) + " dias.")

File: kildey004/programacaobasica/Questao14.java
import java.util.Scanner;

public class Questao14 {
    public static void main(String[] args) {
        int dia;
        int ano;
        int mes;
        int dias;
        Scanner s;

        System.out.println("**Calendario de acidentes**");

        s = new Scanner(System.in);
        System.out.println("Digite a quantidade de dias sem acidentes:");
        dia = s.nextInt();

        ano = dia / 360;
        mes = (dia - ano * 360) / 30;
        dias = dia - ano * 360 - mes * 30;

        System.out.println("tempo sem acidentes: " + ano + " anos, " + mes + " meses, " + dias + "dias.");
    }
}
